conf/messages

```diff
@@ -2212,3 +2212,9 @@ enterPostcode.representee.title=Find the UK contact address
 address-select.representee.title=Select the contact address
 address.uk.representee.title=Enter the UK contact address
 nonUkAddress.representee.title=Enter the non-UK contact address
+
+representeeConfirmPerson.title=Check the details we hold for the person
+representeeConfirmPerson.summaryline1=Name
+representeeConfirmPerson.summaryline2=Capital Gains Tax on UK property account number
```

Comment:
wouldn't we need separate labels for the other id types?

Reply:
I could not see any mention and screen on the ticket. So I thought they all refer to the term "account number"

Reply:
I think it's missing from the ticket - that value there only makes sense for CGT references